README.md

```
@@ -1,40 +1,57 @@
 
-#### Default Invoice Template
+Watch a quick demo of EasyInvoicePDF in action to see how easy it is to create professional invoices in seconds. The video demonstrates key features like **Live Preview**, **Instant PDF Download**, and **Customization Options**.
 
-<img width="1440" height="769" alt="default template" src=".github/screenshots/default.png" />
-
-#### Stripe Invoice Template
-
-<img width="1440" height="769" alt="stripe template" src=".github/screenshots/stripe.png" />
-
-#### Stripe Invoice Template with Logo
+https://github.com/user-attachments/assets/9e3194e0-3072-4c95-ae99-e66db1301dc6
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)